packages/manager/modules/web-paas/src/details/service/service.html

```diff
@@ -0,0 +1,91 @@
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+<div class="row d-lg-flex">
+    <div class="col-md-4">
+        <oui-tile
```

Comment:
`oui-tile` is a block element so
```suggestion
        <oui-tile class="col-md-4"
```